### PR TITLE
Lp 64 app mobile friendly

### LINKED ISSUE #64: Make App mobile friendly
**Is your feature request related to a problem? Please describe.**
When on a smaller screen the app does not scale down.

**Describe the solution you'd like**
On a smaller screen, like a phone, the app should fit the screen based on the width

### PR DESCRIPTION
**Pull request by Lap343**
## Changes
1. make entire app mobile friendly

## Purpose
To have the app scale based on the sizes of the screens

## Approach
setting the background image and setting all components to a fixed position and setting their height and width to %

Closes #64

### CODE REVIEW

**Review by connielion — APPROVED** [review]
Looks good! Oops, I forgot to review with my eyes closed...Did you forget to pull from development before pushing? Looks like there are conflicting files?

File: pokemon-app/src/App.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import Search from "./components/Search";
 import { getEvolutionChain, getPokeByNameOrId } from "./utils/api";
 import "./App.css";
 import "./styles/Pokedex-model.css";
+import "./styles/Pokedex-model-mobile.css";
```
Comment (BlackCubes) [inline pokemon-app/src/App.js:10]: Wat's dis!

**Review by Wolftownn — APPROVED** [review]
Great job working on this. Its coming out insane!

File: pokemon-app/src/components/Search.js [hunk 2]
```diff
@@ -37,79 +50,78 @@ const Search = ({
...
-                Search Enemy
-              </button>
-            </div>
+              Search Enemy
+            </button>
           </div>
         </div>
       </div>
-    </>
+      {pokemon && <ThemeSongs src={source} />}
```
Comment (BlackCubes) [inline pokemon-app/src/components/Search.js:123]: Clean!!!!!!!

File: pokemon-app/src/components/Versus.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import React, { useEffect, useState } from "react";
 import "../styles/Versus.css";
+import "../styles/Versus-mobile.css";
```
Comment (BlackCubes) [inline pokemon-app/src/components/Versus.js:3]: Like the distinction!

**Review by BlackCubes — APPROVED** [review]
A lot of work, but good job!

File: pokemon-app/src/components/Versus.js (hunk 3 above)
Comment (Lap343) [reply pokemon-app/src/components/Versus.js:3]: had to separate. CSS file got to big

**Pull request merged**